python/triton/common/backend.py

```diff
@@ -10,6 +11,9 @@
 
 from ..runtime.driver import DriverBase
 
+TRITON_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+TRITON_VERSION = "2.1.0"
```

Comment:
This seems not optimal to have a hardcoded version string hidden here. Can we have a `__version__` string in `__init__.py`?

Reply:
ACKing that this was already the case before your PR, but maybe it is a good time to change it :p